app/src/main/java/com/intellistart/interviewplanning/slot/application/port/NatsEventPublisherOutPort.kt

```diff
@@ -0,0 +1,8 @@
+package com.intellistart.interviewplanning.slot.application.port
+
+import com.intellistart.interviewplanning.output.pubsub.slot.SlotUpdatedEvent
+
+interface NatsEventPublisherOutPort {
```

Comment:
```suggestion
interface EventPublisherOutPort {
```

No infrastructure details in `*Port` interfaces.
Please, also provide what exactly this out port publishes.